dali/python/nvidia/dali/ops.py

```diff
@@ -27,6 +27,13 @@
 import nvidia.dali.libpython_function_plugin
 
 
+cupy = None
```

Comment:
This is not needed. Bellow global is creating `cupy` variable.

Reply:
When i remove this line, it ends up with:
```
  File "/home/rafal/mydali/build/venv/lib/python3.6/site-packages/nvidia/dali/ops.py", line 33, in _setup_cupy
    if cupy is None:
NameError: name 'cupy' is not defined

```